src/stripe.node.ts

```diff
@@ -0,0 +1,28 @@
+import Stripe = require('./stripe.common');
+import NodePlatformFunctions = require('./platform/NodePlatformFunctions');
+import http = require('http');
+import HttpClient = require('./net/HttpClient');
+
+Stripe._platformFunctions = new NodePlatformFunctions();
+Stripe.webhooks._platformFunctions = Stripe._platformFunctions;
+
+const createNodeHttpClient = (agent: http.Agent): typeof HttpClient => {
```

Comment:
Think about moving this logic into `PlatformFunctions`

Reply:
Agreed, it would be nice to combine this logic all into one place. I'd prefer to do this in a later PR.